app/src/main/java/org/sopt/and/HomeActivity.kt

```diff
@@ -0,0 +1,252 @@
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.home_top_bar),
+            contentDescription = "Top Bar Image",
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(100.dp)
+        )
+    }
+}
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
+    ) {
+        HomeTopBar()
+
+        LazyColumn(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            item {
+                BannerView()
+            }
+
+            item {
+                SectionTitle("믿고 보는 웨이브 에디터 추천작")
+                EditorPicksList()
+            }
+
+            item {
+                SectionTitle("오늘의 TOP 20")
+                Top20List()
+            }
+        }
+    }
+}
+
+@Composable
+fun BannerView() {
+    val images = listOf(
+        R.drawable.banner_image,
+        R.drawable.banner_image,
+        R.drawable.banner_image,
+        R.drawable.banner_image
+    )
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 16.dp)
+    ) {
+        BannerViewPager(images = images)
+    }
+}
+
+@Composable
+fun BannerViewPager(
+    modifier: Modifier = Modifier,
+    images: List<Int>
+) {
+    val pagerState = rememberPagerState(pageCount = { images.size })
+
+    HorizontalPager(
+        state = pagerState,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) { page ->
+        Box(
+            modifier = modifier
+                .fillMaxWidth()
+                .height(450.dp)
+        ) {
+            Image(
+                painter = painterResource(id = images[page]),
+                contentDescription = null,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .fillMaxHeight()
+                    .padding(10.dp),
+                contentScale = ContentScale.Crop,
+                alignment = Alignment.Center
+            )
+            // 현재 페이지 위치 표시
+            Box(
+                modifier = Modifier
+                    .align(Alignment.BottomEnd)
+                    .padding(16.dp)
+            ) {
+                Text(
+                    text = "${page + 1}/${images.size}",
+                    color = Color.White,
+                    modifier = Modifier
+                        .background(
+                            color = Color.Black.copy(alpha = 0.6f),
+                            shape = RoundedCornerShape(12.dp)
+                        )
+                        .padding(horizontal = 12.dp, vertical = 6.dp)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun EditorPicksList() {
+    LazyRow {
+        items(5) {
+            Box(
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(150.dp, 200.dp)
```

Comment:
```suggestion
                    .size(width=150.dp, height=200.dp)
```
가독성이 좋아지는거 같아서 저는 이렇게 named argument 사용하는거 좋아해요!